picard/ui/options/cover_processing.py

```diff
@@ -42,36 +47,133 @@ def __init__(self, parent=None):
-        self.register_setting('resize_images_saved_to_file')
-        self.register_setting('cover_file_maximum_width')
-        self.register_setting('cover_file_maximum_height')
+        self.register_setting('cover_tags_scale_up')
+        self.register_setting('cover_tags_scale_down')
+        self.register_setting('cover_tags_resize_use_width')
+        self.register_setting('cover_tags_resize_target_width')
+        self.register_setting('cover_tags_resize_use_height')
+        self.register_setting('cover_tags_resize_target_height')
+        self.register_setting('cover_tags_stretch')
+        self.register_setting('cover_tags_crop')
+        self.register_setting('cover_file_scale_up')
+        self.register_setting('cover_file_scale_down')
+        self.register_setting('cover_file_resize_use_width')
+        self.register_setting('cover_file_resize_target_width')
+        self.register_setting('cover_file_resize_use_height')
+        self.register_setting('cover_file_resize_target_height')
+        self.register_setting('cover_file_stretch')
+        self.register_setting('cover_file_crop')
+
+        tooltip_keep = N_(
+            "<p>"
+            "Scale the source image so that it fits within the target dimensions.<br />"
+            "One of the final image dimensions may be less than the target dimension if "
+            "the source image and target dimensions have different aspect ratios.<br />"
+            "<br />"
+            "For example, a 2000x1000 image resized to target dimensions of "
+            "1000x1000 would result in a final image size of 1000x500."
+            "</p>"
+        )
+        self.ui.tags_keep.setToolTip(_(tooltip_keep))
+        self.ui.file_keep.setToolTip(_(tooltip_keep))
+        tooltip_crop = (
+            "<p>"
+            "Scale the source image so that it completely fills the target dimensions "
+            "in both directions.<br />"
+            "If the source image and target dimensions have different aspect ratios"
+            "then there will be overflow in one direction which will be (center) cropped.<br />"
+            "<br />"
+            "For example, a 500x1000 image resized to target dimensions of "
+            "1000x1000 would first scale up to 1000x2000, then the excess height "
+            "would be center cropped resulting in the final image size of 1000x1000."
+            "</p>"
+        )
+        self.ui.tags_crop.setToolTip(_(tooltip_crop))
+        self.ui.file_crop.setToolTip(_(tooltip_crop))
+        tooltip_stretch = (
+            "<p>"
+            "Stretch the image to exactly fit the specified dimensions, "
+            "distorting it if necessary.<br />"
+            "<br />"
+            "For example, a 500x1000 image with target dimension of 1000x1000 "
+            "would be stretched horizontally resulting in the final image "
+            "size of 1000x1000."
+            "</p>"
+        )
```

Comment:
A minor suggestion but rather than separating the text using `<br />` you might consider using `</p><p>` which will provide a slight spacing between the sections.  It's a personal preference, but I find it easier to read that way.